pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -143,33 +142,37 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 		}
 		// the status map doesn't exist yet, watching below is still an OK thing to do
 	}
+	originalNodes := 0
+	remainingNodes := util.NewSet()
+	checkStatus := func() {
+		remainingNodes = util.NewSet()
+		// check the nodes to see which ones are already completed
+		originalNodes = len(statuses.Items)
```

Comment:
the 2nd time this function is called, after the watcher results channel is closed, this originalNodes count will have changed because we delete each node's status configmap as they complete.  I believe this originalNodes variable  is only used for logging instead of logic/decisions, so it's not a huge deal, but the logging will look weird.  I think it would look something like:
```
7/10 node(s) completed osd provisioning, resource version 123
orchestration status config map result channel closed, will restart watch
0/3 node(s) completed osd provisioning, resource version 456
```
Let me know if i'm understanding the flow incorrectly.